boogie-conformance/src/main/java/org/mitre/tdp/boogie/conformance/alg/assign/score/FmFeatureExtractor.java

```diff
@@ -0,0 +1,92 @@
+package org.mitre.tdp.boogie.conformance.alg.assign.score;
+
+import static org.apache.commons.math3.util.FastMath.abs;
+import static org.mitre.tdp.boogie.conformance.alg.assign.score.MissingRequiredFieldException.supplier;
+
+import java.util.function.Supplier;
+
+import org.apache.commons.math3.util.FastMath;
+import org.mitre.caasd.commons.HasPosition;
+import org.mitre.caasd.commons.Spherical;
+import org.mitre.tdp.boogie.ConformablePoint;
+import org.mitre.tdp.boogie.Fix;
+import org.mitre.tdp.boogie.MagneticVariation;
+import org.mitre.tdp.boogie.conformance.alg.assign.FlyableLeg;
+import org.mitre.tdp.boogie.viterbi.ViterbiFeatureVector;
+import org.mitre.tdp.boogie.viterbi.ViterbiFeatureVectorExtractor;
+
+public final class FmFeatureExtractor implements Supplier<ViterbiFeatureVectorExtractor<ConformablePoint, FlyableLeg>> {
+
+  /**
+   * Singleton instance of an extractor for shared use.
+   */
+  public static final FmFeatureExtractor INSTANCE = new FmFeatureExtractor();
+
+  /**
+   * Feature who's presence indicates the {@link ViterbiFeatureVector} is associated with an FM leg.
+   */
+  public static final String LEG_TYPE = "IsFm";
+  /**
+   * Feature representing the course delta between the FM declared course and the actual heading of the aircraft.
+   */
+  public static final String DEGREES_OFF_COURSE = "DegreesOffCourse";
+  /**
+   * Feature representing the distance to the FM's fix.
+   */
+  public static final String DISTANCE_FROM_FIX = "DistanceFromFix";
+
+  /**
+   * Feature representing how far before the FM's fix this is. Since the leg includes a true course checking this works.
+   */
+  public static final String DISTANCE_BEFORE_FIX = "DistanceBeforeFix";
+
+  private FmFeatureExtractor() {
+  }
+
+  @Override
+  public ViterbiFeatureVectorExtractor<ConformablePoint, FlyableLeg> get() {
+    return ViterbiFeatureVectorExtractor.<ConformablePoint, FlyableLeg>newBuilder()
+        .addFeatureExtractor(LEG_TYPE, (c, l) -> 1.)
```

Comment:
I don't understand why it is useful to return `1d` all the time - is it different for other legs